app/src/main/java/ai/elimu/appstore/util/InstallationHelper.kt

```diff
@@ -14,6 +15,7 @@ object InstallationHelper {
         try {
             packageManager.getPackageInfo(packageName, PackageManager.GET_ACTIVITIES)
         } catch (e: PackageManager.NameNotFoundException) {
+            Timber.e(e)
```

Comment:
@tuancoltech Please don't print this as an error. It's confusing to see error messages in the log during debugging when there isn't any error happening. 

An application to not be installed is expected, so I suggest we either remove this code change, or change the log level to `WARNING` or lower.